.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
@psquickitjayant could you follow-up with me offline about the account credentials? You work at Loggly right? If Loggly can provide credentials to a free account for this module I think that would fix the problem.